ui/src/components/Rules/RuleGroup/index.tsx

```diff
@@ -47,72 +47,76 @@ const RuleGroup = (props: {
   }
 
   return (
-    <div className="flex-auto">
-      <div className="text-base font-bold text-white sm:text-lg">
-        {props.group.name}
-      </div>
-      <div className="mt-2 h-28 overflow-hidden overflow-y-auto overflow-ellipsis font-semibold">
-        {props.group.description}
+    <>
+      <div className="inset-0 z-0 h-fit p-3">
+        <div className="overflow-hidden overflow-ellipsis whitespace-nowrap text-base font-bold text-white sm:text-lg">
+          <div>{props.group.name}</div>
+        </div>
+        <div className="h-12 max-h-12 overflow-y-hidden whitespace-normal text-base text-zinc-400 hover:overflow-y-scroll">
+          {props.group.description}
+        </div>
       </div>
-      <div className="mb-2 mt-2 grid grid-cols-3 gap-3">
-        <div>
-          <div className="align-center flex justify-center text-sm font-medium">
-            Status
-          </div>
+      <div className="inset-0 z-0 h-fit p-3">
+        <div className="mb-5 mt-5 grid grid-cols-3 gap-3">
           <div>
-            {props.group.isActive ? (
-              <span className="flex justify-center font-semibold text-green-600">
-                Active
-              </span>
-            ) : (
-              <span className="flex justify-center font-semibold text-red-600">
-                Inactive
-              </span>
-            )}
+            <div className="align-center flex justify-center font-bold">
+              Status
+            </div>
+            <div>
+              {props.group.isActive ? (
+                <span className="flex justify-center text-green-600">
+                  Active
+                </span>
+              ) : (
+                <span className="flex justify-center text-red-600">
+                  Inactive
+                </span>
```

Comment:
The collection page uses text-green-500 and text-red-500 for these. Can we align them?